content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: delfin
+description: "delfin deployment on kubernetes guide."
```

Comment:
correct it:
delfin deployment on kubernetes cluster

Reply:
corrected